system-addon/lib/SectionsManager.jsm

```diff
@@ -41,7 +41,8 @@ const BUILT_IN_SECTIONS = {
       button: {id: options.disclaimer_buttontext || "section_disclaimer_topstories_buttontext"}
     },
     maxRows: 1,
-    availableContextMenuOptions: ["CheckBookmark", "SaveToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl"],
+    sectionMenuOptions: ["RemoveSection", "CheckCollapsed", "Separator", "AboutPocket", "ManageSection"],
```

Comment:
I wonder if we want to have a common list where "About" gets added in dynamically?

Reply:
I guess we could have a `"CheckAbout"`. I'll try that.